testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -197,6 +197,147 @@ public void testReadSparkCreatedTable(String sparkTableFormat, String expectedTr
         };
     }
 
+    @Test(groups = {HIVE_SPARK, PROFILE_SPECIFIC_TESTS})
+    public void testReadTrinoCreatedOrcTable()
+    {
+        testReadTrinoCreatedTable("using_orc", "ORC");
+    }
+
+    @Test(groups = {HIVE_SPARK, PROFILE_SPECIFIC_TESTS})
+    public void testReadTrinoCreatedParquetTable()
+    {
+        testReadTrinoCreatedTable("using_parquet", "PARQUET");
+    }
+
+    @Test(groups = {HIVE_SPARK, PROFILE_SPECIFIC_TESTS})
+    public void testReadTrinoCreatedParquetTableWithNativeWriter()
+    {
+        onTrino().executeQuery("SET SESSION " + TRINO_CATALOG + ".experimental_parquet_optimized_writer_enabled = true");
+        // TODO (https://github.com/trinodb/trino/issues/7953) Native Parquet Writer writes Parquet V2 files that are not compatible with Spark's vectorized reader
```

Comment:
```suggestion
        // TODO (https://github.com/trinodb/trino/issues/6377) Native Parquet Writer writes Parquet V2 files that are not compatible with Spark's vectorized reader, see https://github.com/trinodb/trino/issues/7953 for more details
```

Reply:
I'd rather just get some version of this merged so that we can fix the remaining issues. I've addressed this and pushed the newest version. 

@findepi, @hashhar can one of you merge?